bitnami/redis/templates/redis-master-statefulset.yaml

```
@@ -231,7 +231,7 @@ spec:
               containerPort: 9121
           resources: {{- toYaml .Values.metrics.resources | nindent 12 }}
   {{- end }}
-      {{- $needsVolumePermissions := and .Values.volumePermissions.enabled (and ( and .Values.master.persistence.enabled (not .Values.persistence.existingClaim) ) .Values.securityContext.enabled) }}
+        {{- $needsVolumePermissions := and .Values.volumePermissions.enabled (and ( .Values.master.persistence.enabled ) .Values.securityContext.enabled) }}
```

It can be simplified:

```suggestion
      {{- $needsVolumePermissions := and .Values.volumePermissions.enabled .Values.master.persistence.enabled .Values.securityContext.enabled }}
```